### PR TITLE
Add session guard + stuff

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: libs/event/src/lib/layout/view/view.component.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ export class EventViewComponent implements OnInit {
   ngOnInit(): void {
     this.invitation$ = this.event$.pipe(
       switchMap(event => this.invitationQuery.selectByDocId(event.id)),
-    )
+      shareReplay(1)
```
Comment (phpgeekfr) [inline libs/event/src/lib/layout/view/view.component.ts:34]: what is this ?

File: libs/event/src/lib/pipes/event-time.pipe.ts [hunk 2]
```diff
@@ -1,18 +1,25 @@
 import { Pipe, PipeTransform, NgModule } from '@angular/core';
 import { Event } from '../+state/event.model';
 
+export type EventTime = 'early'|'onTime'|'late';
```
Comment (phpgeekfr) [inline libs/event/src/lib/pipes/event-time.pipe.ts:4]: linter

File: libs/ui/src/lib/app-bar.ts [hunk 3]
```diff
@@ -123,6 +128,7 @@ export class PageBarDirective implements AfterViewInit, OnDestroy {
   ngOnDestroy() {
     this.appContainer.appBar.detach();
     if (this.targetId && this.observer) {
+      console.log('Unobserve');
```
Comment (phpgeekfr) [inline libs/ui/src/lib/app-bar.ts:131]: forgot to remove ?

File: libs/event/src/lib/layout/view/view.component.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply libs/event/src/lib/layout/view/view.component.ts:34]: To share the result multiple time. else each "| async" will call the database.

**Pull request merged**